core/src/schema/content/number.rs

```diff
@@ -491,6 +494,7 @@ impl number_content::I64 {
                     Ok(number_content::F64::Constant(cast).into())
                 }
             },
+            I64::Id(_) => unimplemented!()
```

Comment:
I think this should only be called when `OptionalMergeStrategy` is used, which as of now happens in a rather strange way when doing a database import (at the point of sampling). Are we sure this won't panic imports?